Add String Wildcard section (#10)
SigmaHQ/sigma-specification: Sigma_1_0_1.md

* Regular expressions are case-sensitive by default
* You don't have to escape characters except the string quotation marks `'`

#### String Wildcard

> Wildcard are used when a part of the text is random.  
> You can use :
> 
> * `?` replace a single mandatory character 
> * `*` replace an unbounded length wildcard

Reviewer: ```suggestion
Wildcards are used when part of the text is random.  
You can use :

* `?` to replace a single mandatory character 
* `*` to replace an unbounded length wildcard
```

example  :

* `progA.exe or progB.exe or ...` will be `prog?.exe`
* `antivirus_V1.exe or antivirus_V21.2.1.exe or ...` will be `antivirus_V*.exe`

> Sigma have special modifier to facilitate the search of unbounded string

Reviewer: ```suggestion
Sigma has special modifiers to facilitate the search of unbounded strings
```

> * `*somesthing` see [endswith modifier](#value-modifiers)
> * `somesthing*` see [startswith modifier](#value-modifiers)
> * `*somesthing*` see [contains modifier](#value-modifiers)

Reviewer: ```suggestion
* `*something` see [endswith modifier](#value-modifiers)
* `something*` see [startswith modifier](#value-modifiers)
* `*something*` see [contains modifier](#value-modifiers)
```
